androidNode/src/androidMain/kotlin/network/bisq/mobile/android/node/utils/ImageUtils.kt

```diff
@@ -53,8 +54,8 @@ object ImageUtil: Logging {
 
     internal fun getImageByPath(context: Context, path: String): Bitmap? {
         return try {
-            log.d { "opening file in path $path"}
-            val inputStream = context.assets.open(path)
+            val fullPath = "cathash/$path"
```

Comment:
this makes the ImageUtil generic class specific to cat hashing.

Please pass the `cathash` as a baseDir parameter or as part of the path parameter

Reply:
Its a specific ImageUtils for only the cathash use case. We should rename the class to make that more clear. But yes, we can move it as params.

As there will be bigger changes anyway as we need to generate the images also for client mode I would prefer to skip that for later. Probably that class will move to a KMP compatible version then.

Reply:
Done